src/main/java/team/baymax/logic/commands/patient/RemarkPatientCommand.java

```diff
@@ -18,7 +18,7 @@
  */
 public class RemarkPatientCommand extends Command {
 
-    public static final String COMMAND_WORD = "remark";
+    public static final String COMMAND_WORD = "remarkpat";
```

Comment:
shotening to "pat" from "patient" could be confusing for the user as "pat" is not a conventional short form for patient (unlike appt)